modules/util/portal-tools-rest-builder/build.gradle

```diff
@@ -22,19 +22,28 @@ dependencies {
 	api group: "com.puppycrawl.tools", name: "checkstyle", version: "8.29"
 	api group: "commons-io", name: "commons-io", version: "2.15.1"
 	compileInclude group: "com.beust", name: "jcommander", version: "1.82"
+	compileInclude group: "com.liferay", name: "com.liferay.portal.vulcan.api", version: "44.1.0"
```

Comment:
Hey @drewbrokke,

With these changes we will not be compiling against the source code anymore and thus we will not be able to use new features if needed.

Maybe we need to set a cut point and not support java8 from now on with REST Builder and provide a java8 compatible version for customers that need it, which is built from the last java8 compatible point of master and be frozen.

Reply:
See https://github.com/liferay-headless/liferay-portal/pull/2487 where we are trying to get rid of the vulcan-api dependency.